<launch>
    <node pkg="ros_tutorial" type="obstacle.py" name="obstacle" output="screen"></node>

    <node pkg="ros_tutorial" type="vfh_algo.py" name="vfh_algo" output="screen">
       <param name="goal_x" value="$(arg goal_x)" />
       <param name="goal_y" value="$(arg goal_y)" />

    </node>

    <node pkg="ros_tutorial" type="monitor.py" name="monitor"></node>
    
    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_funky_maze.launch">
        <arg name="x_pos" value="$(arg x_pos)" />
        <arg name="y_pos" value="$(arg y_pos)"/>
        <arg name="z_pos" value="0.0"/>
        <arg name="yaw" value="1.57075"/>
    </include>

    <include file="$(find turtlebot_rviz_launchers)/launch/view_robot.launch"/>

   
</launch>